rust/rust.bzl

```diff
@@ -629,15 +663,14 @@ _rust_toolchain_attrs = {
 
 _rust_library_attrs = _rust_common_attrs + {
     "crate_type": attr.string(),
+    "platform": attr.string(),
 }
 
 rust_library = rule(
     _rust_library_impl,
     attrs = _rust_library_attrs + _rust_toolchain_attrs,
     fragments = ["cpp"],
-    outputs = {
-        "rust_lib": "lib%{name}.rlib",
-    },
+    outputs = _get_crate_type_and_target_outputs,
```

Comment:
This is a problem on our side, that should be documented. I believe @fweikert is the one who implemented the callback for outputs.